resources/views/landing_about.edge

```diff
@@ -5,6 +5,14 @@
 <!--[if IE 9]>         <html lang="en" class="no-js ie9 lt-ie10"> <![endif]-->
 <!--[if gt IE 9]><!--> <html lang="en" class="no-js"> <!--<![endif]-->
 <head>
+  <!-- Google Tag Manager -->
+  <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+      new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+  })(window,document,'script','dataLayer','GTM-P272BJ4J');</script>
```

Comment:
@johnnycho due this is a open source project, I would recommend to inject this as an optional and "dynamic" variable.

For instance, define the GTM unique identifier in a `.env` variable AND if it's defined, then render the while `<script>` block with the GTM identifier variable inside it.

Reply:
@TonnyORG PR updated.